components/Typography/src/UIFont+MaterialSimpleEquality.h

```diff
@@ -0,0 +1,38 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+/**
+ The Typography component provides methods for getting sized fonts and opacities following Material
+ style guidelines.
+
+ This header is the umbrella header for the component and should be imported by consumers of the
+ Typography component. Please do not directly import other headers. This will allow the componet to
+ expand or contract the header file space without consumer modifications.
+ */
+
+#import <UIKit/UIKit.h>
+
+@interface UIFont (MaterialSimpleEquality)
+
+/*
+ Checks simple characteristics: name, weight, pointsize, traits.
+
+ While the actual implementation of UIFont's isEqual: is not known, it is believed that
+ isEqualToFont: is more 'shallow' than isEqual:.
```

Comment:
Docs look like they need to be updated:

> UIFont's `isEqual:` is more 'shallow' than this method's implementation.

Reply:
No, it's correct. Mine only checks 4 properties. I think `isEqual:` checks more.

Reply:
Ah sorry, I should have been more clear that the “isEqualToFont:” reference is, I believe, no longer applicable.

Reply:
Ah sorry, I should have been more clear that the “isEqualToFont:” reference is, I believe, no longer applicable.

Reply:
Oh oh oh. Sorry. yes. Thank you. Done!